packages/builder/src/pages/builder/Branding.svelte

```diff
@@ -8,7 +8,7 @@
   $: platformTitle =
     !$auth.user && platformTitleText ? platformTitleText : "Budibase"
 
-  $: faviconUrl = $organisation.faviconUrl || "https://i.imgur.com/Xhdt1YP.png"
+  $: faviconUrl = $organisation.faviconUrl || "/builder/bblogoimgur.png"
```

Comment:
Should we remove the `imgur` from this - since we don't need to worry about imgur anymore, might be better to just reference any reference to the fact we ever used it.

Reply:
Yeah, it's kind of redundant - mainly just used it to keep myself right when I was replacing the imgur URLs. Will update!